src/main/IO/Process/Services/IdentityAccess/RegionPermitRepository.cs

```diff
@@ -23,25 +23,35 @@ public RegionPermitRepository(IAvatarContextService avatarContextService)
         this.avatarContextService = avatarContextService;
     }
 
-    public async Task DeleteAsync(RegionPermit regionPermit)
+    public async Task<RegionPermit> GetByIdAsync(int sequenceId)
```

Comment:
This method is not used but I didn't delete it since it might be useful later on.